server/mergin/auth/utils.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) Lutra Consulting Limited
+#
+# SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
+
+import re
+
+
+def contains_email_invalid_characters(email: str) -> bool:
+    """Check for invalid characters for the email address"""
+    invalid_characters = r"[ ;:’\–\—|,]"
```

Comment:
What about two `.` in username, how is this handled?

Do we have some sources about 'standard' emails validations?